.for_maintainers/release/AMI_RELEASE.md

```diff
@@ -0,0 +1,101 @@
+# Releasing the publicly-accessible Wallaroo Amazon Machine Images (AMIs)
+
+This document is aimed at members of the Wallaroo team who might be releasing
+AMIs for a release candidate or release branch.
+
+To learn more about our release process, see [RELEASE.md](RELEASE.md).
+
+## Prerequisites for releasing the AMIs
+
+The AMI building and publishing process is conducted inside a Vagrant
+machine. You'll therefore need:
+
+* Vagrant installed
+* A provisioned Wallaroo Release Vagrant box, as described in
+  [PROVISION_VAGRANT.md](PROVISION_VAGRANT.md).
+* Valid AWS credentials in `~/.aws`
+
+## Releasing the AMIs
+
+### Start up Vagrant box
+
+```bash
+cd .release
+DOCKER_USERNAME=<YOUR_USERNAME> \
+DOCKER_PASSWORD=<YOUR_BINTRAY_DOCKER_API_ACCESS_KEY> \
+DOCKER_SERVER=wallaroo-labs-docker-wallaroolabs.bintray.io vagrant up
+```
+
+Also: please be aware that your local directory `~/.aws` is mounted inside the
+Vagrant box under `/home/vagrant/.aws`.
+
+
+### SSH into Wallaroo Vagrant box
+
+From within the `.release` directory run:
+
+```bash
+vagrant ssh
+```
+
+This will `ssh` you into the running Wallaroo box.
+
+### Build and publish AMI of your selected Wallaroo version
+
+```bash
+./ami-release.sh <BRANCH> <TARGET_AMI_VERSION>
```

Comment:
can this be updated to be run from the root wallaroo directory, as opposed to the `.release` directory? so it can be run like the other scripts? currently will not run from the root directory